pkgs/development/python-modules/pygame-ce/default.nix

```diff
@@ -19,9 +19,6 @@
   libX11,
   portmidi,
   SDL2_classic,
-  SDL2_classic_image,
-  SDL2_classic_mixer,
-  SDL2_classic_ttf,
   numpy,
```

Comment:
rather than remove the SDL2_classic_* inputs my intention with suggesting this passthru.sdlLibs pattern is for `inherit (SDL2_classic) SDL2_image ...;` to go in the `callPackage` expressions in `pkgs/top-level/python-packages.nix`